modules/tinymce/src/core/main/ts/Rtc.ts

```diff
@@ -251,15 +251,22 @@ const makeNoopAdaptor = (): RtcAdaptor => {
 
 export const isRtc = (editor: Editor) => Obj.has(editor.plugins, 'rtc');
 
+const getRtcSetup = (editor: Editor): Optional<() => Promise<RtcRuntimeApi>> => {
+  return (Obj.get(editor.plugins, 'rtc') as Optional<RtcPluginApi>).bind((rtcPlugin) => {
+    // This might not exist if the stub plugin is loaded on cloud
+    return Optional.from(rtcPlugin.setup);
+  });
+};
```

Comment:
```suggestion
const getRtcSetup = (editor: Editor): Optional<() => Promise<RtcRuntimeApi>> =>
  Obj.get(editor.plugins, 'rtc') as Optional<RtcPluginApi>).bind((rtcPlugin) => {
    // This might not exist if the stub plugin is loaded on cloud
    return Optional.from(rtcPlugin.setup);
  });
```

We can use the shorthand for this, which arguably helps make it a little easier to read as well :shrug:

Reply:
why not shorthand both 🤷‍♂️

Reply:
I did consider that, but I assumed that was done purposely :shrug: if that's not the case then yeah we might as well

Reply:
I updated the suggestion, as it probably wasn't done purposely thinking more.